From: Modestas Vainius <modax@debian.org>
Subject: Enable DLRestrictions (REJECT:OTHERSOVERSION) support for all libraries
Forwarded: not-needed
Origin: vendor
Last-Update: 2011-08-11

Enable REJECT:OTHERSOVERSION for these libraries because their ABI is typically
not stable and structures frequently conflict.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -264,3 +264,10 @@ install(EXPORT kdeworkspaceLibraryTarget
 
 # Add support for DebianABIManager
 include(/usr/share/pkg-kde-tools/cmake/DebianABIManager.cmake)
+
+# Enable DLResctrictions
+if (CMAKE_BUILD_TYPE STREQUAL "Debian")
+    find_package(DLRestrictions)
+    set(DEFAULT_DLRESTRICTIONS "REJECT:OTHERSOVERSION")
+    kde4deb_dlrestrictions_process_libraries()
+endif (CMAKE_BUILD_TYPE STREQUAL "Debian")
